libcxx/utils/ci/Dockerfile

```diff
@@ -5,11 +5,29 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 #
 #===----------------------------------------------------------------------===##
-
 #
```

Comment:
Not attached to this line: What testing did you do? Did you build the images and run some of the CI jobs to make sure things won't blow up when we change to these images?

Reply:
I did. I tested (and am using) the `testing` image for the github actions runners, and I ran a few jobs on the buildkite ones (which initially discovered some now addressed issues).